Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs

```diff
@@ -69,13 +71,44 @@ public async Task<IActionResult> RegisterUserAsync(AddUserDTO user)
             return Ok(response.Message);
         }
 
+        [HttpPost]
+        [Authorize(Roles = "User")]
+        [Route("/NewPayment/{userId}")]
```

Comment:
you can specify route within `[HttpPost]` attribute

Reply:
endpoint route moved to [HttpPost] attribute